cmd/kubectl-gadget/utils/errors.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Gadget pod
+var (
+	ErrGadgetPodNotFound      = errors.New("gadget pod not found")
+	ErrMultipleGadgetPodFound = errors.New("multiple gadget pods found")
+)
+
+// Kubernetes client
+func WrapInErrSetupK8sClient(err error) error {
+	return fmt.Errorf("failed to set up Kubernetes client: %w", err)
+}
+
+func WrapInErrListNodes(err error) error {
+	return fmt.Errorf("failed to list nodes: %w", err)
+}
```

Comment:
I'm wondering if we could use a different approach. 
1. Define the error messages as strings on this file. `var ListNodesErrMsg = "failed to list nodes"`
2. Users can use `errors.Wrap(err, utils.ListNodesErrMsg)` instead of `WrapInErrListNodes()`

I'm not sure it's much better, perhaps resulting code is easier to follow as there is not a call to a function?

Reply:
Forget about it, I was looking at https://pkg.go.dev/github.com/pkg/errors#Wrap but it's not the standard package...